modules/core/src/v2/coins/trx.ts

```diff
@@ -192,4 +205,47 @@ export class Trx extends BaseCoin {
 
     return Buffer.from(sig, 'hex');
   }
+
+  /**
+   * Explain a Tron transaction from txHex
+   * @param params
+   * @param callback
+   */
+  explainTransaction(
+    params: ExplainTransactionOptions,
+    callback?: NodeCallback<TransactionExplanation>
+  ): Bluebird<TransactionExplanation> {
+    return co<TransactionExplanation>(function*() {
+      const txHex = params.txHex || (params.transaction && params.transaction.txHex);
+      if (!txHex || !Tron.safeTronHexString(txHex)) {
+        throw new Error('missing valid txHex string');
+      }
+      if (!params.fee) {
+        throw new Error('missing fee');
+      }
+      if (!params.txID) {
+        throw new Error('missing txID');
+      }
+      const rawContract = Tron.getRawTransferContract(txHex);
+      const transfer = Tron.decodeTransferContract(rawContract.contractList[0].parameter.value);
+      const outputAmount = transfer.amount;
+      const outputs = {
+        amount: outputAmount,
+        address: transfer.toAddress, // Should turn it into a readable format, aka base58
+      };
+      return {
+        displayOrder: ['id', 'outputAmount', 'changeAmount', 'outputs', 'changeOutputs', 'fee', 'memo'],
+        id: params.txID,
+        outputs,
+        outputAmount,
+        changeOutputs: [], // account based does not use change outputs?
+        changeAmount: 0, // account base does not make change
+        fee: params.fee,
+        expiration: rawContract.expiration,
```

Comment:
These are not contained in the base TransactionExplanation -- we can make a TronTransactionExplanation interface that extends the base interface, containing these tron-specific fields

Reply:
This is what the account lib is supposed to abstract from us. I'll add a get field for this.

Reply:
Did that on line 43 :D